hyp3_gamma/metadata/README.md

```diff
@@ -0,0 +1,133 @@
+# HyP3 GAMMA Metadata Templates
+
+Package for generating HyP3 GAMMA products' metadata.
+
+The goal is for each product to have a README text file, which gives an overview
+of the product, including a list of all the files contained in the product zip
+file with brief summaries for each file. In addition, some products have
+individual xml files for each of the raster files contained in the product
+archive. They have been designed and formatted to display correctly in the
+ArcGIS metadata environment (Item Description in ArcGIS Desktop, and Metadata
+tab in ArcGIS Pro).  See the "ArcGIS-Compatible Metadata" section below for
+formatting tips.
+
+## Generating An Example Set
+
+Because we use `jinja2` templates to generate the product metadata files, it's
+not possible to preview your changes live, especially in an ArcGIS environment.
+To preview your changes, you can generate an example set of metadata for an RTC
+or InSAR product by running either:
+
+```
+python -m hyp3_gamma.metadata rtc
+python -m hyp3_gamma.metadata insar
+```
```

Comment:
@hjkristenson moving hyp3-metadata-templates inside gamma will mean you'll run it with `hyp3_gamma.metadata` instead of just `hyp3_metadata`.We'll help you get a `hyp3-gamma` development env setup.